Sources/Cells/UserAdTableViewCell/UserAdTableViewCell.swift

```diff
@@ -0,0 +1,224 @@
+//
+//  Copyright © 2020 FINN AS. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+public class UserAdTableViewCell: UITableViewCell {
+
+    // MARK: - Public properties
+
+    public enum Style {
+        case `default`
+        case compressed
+    }
+
+    public var remoteImageViewDataSource: RemoteImageViewDataSource? {
+        didSet {
+            adImageView.dataSource = remoteImageViewDataSource
+        }
+    }
+
+    public var loadingColor: UIColor? = .toothPaste
```

Comment:
Doesn't seem like we're doing much with this rather than using it as `.toothPaste`, but this is perhaps something we want to consider?

Reply:
It's set from the data source in the ad-insertion module.